dragonchain/lib/segwit_addr.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2017 Pieter Wuille
```

Comment:
What's this?

Reply:
A vendored dependency

Reply:
Oh this was the file we needed, but we didn't want the whole dependency. I remember now.